### ISSUE TITLE
NTR: Carboxysterols high in cultured lymphoblasts (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8279]
New term request Carboxysterols high in cultured lymphoblasts (elthat_110328055003):

**Comment by pnrobinson** [comment]
@MickeySegal -- similar question as in #8278

**Comment by MickeySegal** [comment]
Also from https://www.ncbi.nlm.nih.gov/books/NBK51754/ "When cultured in cholesterol-depleted medium, lymphoblastoid cells of individuals with CHILD syndrome and CK syndrome have increased levels of methyl- and carboxy-sterols and slightly decreased levels of cholesterol".

This seems more compelling than the "slightly decreased levels of cholesterol" part, but if we decide this is clinically useful it seems we would need to specify "cultured in cholesterol-depleted medium".

**Comment by pnrobinson** [comment]
@MickeySegal 
this seems like a research-only assay, and as such is outside the scope.

**Comment by MickeySegal** [comment]
GeneReviews doesn't tend to include research-only tests in "Analyte testing".

**Comment by pnrobinson** [comment]
@MickeySegal  thanks, adding!
Adding new term: 
- ID: HP:6001254
- label: Elevanted carboxysterol concentration in cultured lymphoblasts
- parents: [Abnormal tissue metabolite concentration (HP:0032243)](https://hpo.jax.org/app/browse/term/HP:0032243)
- pmid: PMID:21290788
- def: The concentration of carboxysterols in lymphoblastoid cells cultured in cholesterol-depleted medium is below the lower limit of normal.
- comment: When cultured in cholesterol-depleted medium, lymphoblastoid cells of individuals with CHILD syndrome and CK syndrome have increased levels of methyl- and carboxysterols and slightly decreased levels of cholesterol.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8279
- synonyms:
